test/units/template/test_templar.py

```diff
@@ -341,6 +342,15 @@ def test_lookup_jinja_dict_list_passed(self):
                                 'dict',
                                 ['foo', 'bar'])
 
+    def test_lookup_jinja_list_dict_passed(self):
+        try:
```

Comment:
Rather than using `try... except`, please use one of the `unittest` methods, such as `assertRaises` or `assertRaisesRegexp`. See surround tests for examples.

Reply:
I thought so too.
```
    def test_lookup_jinja_list_dict_passed(self):
        self.assertRaisesRegexp(AnsibleError,
                                "with_list expects a list",
                                self.templar._lookup,
                                'list',
                                {'foo': 'bar'})
```
```
$ ansible-test units --tox --python 2.7 test/units/template/test_templar.py
```

But it does not work.
```
   def test_lookup_jinja_list_dict_passed(self):
        self.assertRaisesRegexp(AnsibleError,
                                "with_list expects a list",
                                self.templar._lookup,
                                'list',
>                               {'foo': 'bar'})
E       AssertionError: AnsibleError not raised
```